### PR TITLE
Add more info to make life easier for users

### PR DESCRIPTION
**Pull request by votdev**
While trying to setup a Harvester environment using the Vagrant solution, i came across several issues that i want to improve with this issue to make it other users easier.

### CODE REVIEW

**Review by irishgordo — APPROVED** [review]
:+1: looks good to me :smile: 
lgtm!

File: vagrant-pxe-harvester/README.md [hunk 1]
```diff
@@ -17,22 +17,29 @@ Prerequisites
 -   vagrant-libvirt plugin \>= 0.0.43.
 -   KVM (i.e. qemu-kvm), preferably the latest and greatest. This
     environment was tested with qemu-kvm 2.11.
--   Host with at least 16 CPU, 64GB RAM, and 500GB free disk space.
+-   Host with at least 16 CPU, 64GB RAM, and 500GB free disk space. (note: running this is **very** resource intensive depending on the number of nodes)
```
Comment (irishgordo) [inline vagrant-pxe-harvester/README.md:20]: :smile: I like the emphasis on: "**very**" :smile: :+1: 
As Harvester absolutely is when trying to run a full cluster virtualized, and it's great to mention that.

**Review by slickwarren — APPROVED** [review]
lgtm

File: vagrant-pxe-harvester/README.md [hunk 2]
```diff
@@ -17,22 +17,29 @@ Prerequisites
...
 
 1.  Edit `settings.yml` to make sure the configuration satisfies your
     needs. The options are self-documented.
-2.  Run `setup_harvester.sh`. This may take awhile (i.e. 30 minutes
-    depending on configuration).
+    Set `harvester_cluster_nodes` to `1` if you just want to try out
+    Harvester quickly and don't need a full cluster.
+2.  Run `setup_harvester.sh` to deploy a Harvester cluster only.
+    This may take a while (i.e. 30 minutes  depending on configuration).
+    To deploy a Harvester cluster and a Rancher node, run `vagrant up`.
```
Comment (irishgordo) [inline vagrant-pxe-harvester/README.md:31]:
```suggestion
    To deploy a Harvester cluster and a Rancher node, [enable Rancher](https://github.com/harvester/ipxe-examples/blob/08a7053e68e7983e3d1f6c776823f45c0b170261/vagrant-pxe-harvester/settings.yml#L141).
```

**Review by irishgordo — CHANGES_REQUESTED** [review]

File: vagrant-pxe-harvester/README.md (hunk 2 above)
Comment (votdev) [reply vagrant-pxe-harvester/README.md:31]: I've opened https://github.com/harvester/ipxe-examples/pull/74 to fix that.

**Review by irishgordo — APPROVED** [review]

**Review by khushboo-rancher — APPROVED** [review]

**Pull request merged**